### ISSUE TITLE
Increasing required rank for requesting Air assets 

### ISSUE THREAD

**Issue by HamzaXSniper** [UncreatedStaff/UncreatedWarfare#80]
its not realistic to let the most incompetent players fly a jet and trying to shoot an bradley with vymples rather than using the designated laser seat and ending up dead,
Most new players take out a jet to do one out of these three 
1.Kamikaze
2.Doing the stupidest shit you have ever seen due to not understanding the basic mechanics of the server and ruining the rest of the match.
3.Misusing the air vehicles for "transport" or soloing

I suggest adding a simple rank increase to the minimum level required for the jet, i understand that the level was set low to give players a reachable goal and let everyone "have a go" at air vehicles. But at this point it's very easy to reach that goal by alot of different methods including exp farming. I understand that this minimum rank increase will also consequently increase the amount of people exp farming, But i have seen very promising fixes from issue #66 . 

I would love to see what other players have to say about this issue and other possible fixes.

**Comment by MissileMagnet01** [comment]
I agree with what you're saying, new players might not even know what an AA (Igla/Stinger) is, increasing the rank requirements for air vehicles might just fix the problem.

**Comment by 3312bKarp** [comment]
I'd say that most of the people that try* to fly jet/attack heli know how jets aa flying etc work, but they simply don't have experience playing them. And raising the rank won't change them being not experienced. I think it's more of a trial and error until they learn from their mistakes or get a different place to train on like a training server.

**Comment by MissileMagnet01** [comment]
For the trial and error method to work you would need to either crash or get shot down by AA missiles, which makes it abit slightly harder for the team to win, I think having the whole team suffer just because a player needs practise on piloting air vehicles is kinda unfair.

**Comment by HamzaXSniper** [comment]
I see your point about unexperience but there is definitely a line between unexeperience and just wack shit crazy and asset spamming, raising the rank minimum will force the players to play more and hopefully gain more experience and learn from other's mistakes and successes.

**Comment by HamzaXSniper** [comment]
I see your point about unexperience but there is definitely a line between unexeperience and just wack shit crazy and asset spamming, raising the rank minimum will force the players to play more and hopefully gain more experience and learn from other's mistakes and successes.

**Comment by 3312bKarp** [comment]
Re: hamza
raising the level could throw out some of the more retarded players but it would be a small chunk
Re: Missile magner
but there is no other place to practice jets except flying in single which doesn't give you too much
It's a paradox were to fly the jet you need to git gud but to git gud you need to fly the jet.

**Comment by UsernameIsUnavailible** [comment]
faq 16

**Comment by UsernameIsUnavailible** [comment]
Anyways, The real reason why its so "low" (atleast for the beginning) was to get players have a go at the air assets early on rather then putting a wall behind it so you were forced to grind XP for 3 weeks straight). But yeah must agree that some idiots can ruin your match if they die in air assets

**Comment by PaawllooPL** [comment]
when, dear Var users?
